sql/moz-fx-data-shared-prod/bigeye_derived/dashboard_service_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+friendly_name: Dashboard Service
+description: |-
+  BigEye metadata for the dashboard service
+owners:
+- phlee@mozilla.com
+labels:
+  application: bigeye
+  incremental: false
+  schedule: daily
+  table_type: aggregate
+scheduling:
+  dag_name: bqetl_bigeye_derived
```

Comment:
This DAG needs to be defined inside [dags.yaml](https://github.com/mozilla/bigquery-etl/blob/main/dags.yaml).